### DISCUSSION TITLE
What can cause ` timed out waiting for database to become available` when starting?

### DISCUSSION THREAD

**Discussion by aaadddfgh** [coder/coder#15468] [General]
I am starting coder for windows and get following error:
```
Coder v2.17.0+5a6d23a - Your Self-Hosted Remote Development Platform
failed to sync sloghuman: sync /dev/stderr: The handle is invalid.
Started HTTP listener at http://192.168.1.62:4000
Using built-in PostgreSQL (xxx\AppData\Roaming\coderv2\postgres)
Encountered an error running "coder server", see "coder server --help" for more information
error: Failed to start built-in PostgreSQL. Optionally, specify an external deployment with `--postgres-url`: timed out waiting for database to become available
```

**Comment by matifali** [comment]
Hi, @aaadddfgh, the built-in Postgres is to quickly try Coder and is not recommended for production use. 

Also as per the [docs](https://coder.com/docs/install/cli), you need to install [Visual C++ Runtime](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist#latest-microsoft-visual-c-redistributable-version) to use the builtin Postgres.
